react-delegationdashboard/src/components/Layout/index.tsx

```diff
@@ -2,8 +2,9 @@ import { QueryResponse } from '@elrondnetwork/erdjs/out/smartcontracts/query';
 import denominate from 'components/Denominate/formatters';
 import { denomination, decimals } from 'config';
 import { useContext, useDispatch } from 'context';
+import { emptyAgencyMetaData } from 'context/state';
 import { contractViews } from 'contracts/ContractViews';
-import { ContractOverview, NetworkConfig, NetworkStake, Stats } from 'helpers/types';
+import { ContractOverview, AgencyMetadata, NetworkConfig, NetworkStake, Stats } from 'helpers/types';
```

Comment:
maybe use typescript IBeforeInterface notation to identify types in project. Or NetworkConfigType (suffix with Type at the end)

Reply:
Isn't the same for ```ContractOverview``` ?

Reply:
OK